logproof/src/linear_relation.rs

```diff
@@ -833,13 +833,65 @@ impl LogProof {
         (r_2, r_1)
     }
 
+    /**
+     * Takes an individual values in a field `Zq`, treats the value as signed [q's
+     * complement](https://en.wikipedia.org/wiki/Method_of_complements) and
+     * converts the value to binary 2's complement. This is then appended to an
+     * provided BitVec.
+     *
+     * `value` is the element in Zq and `b` is the number of bits needed
+     * to represent the signed value.
+     *
+     * This modifies bitvec in place.
+     *
+     */
+    #[inline]
```

Comment:
Don't know that this needs to be inline but :shrug:

Reply:
Ya I was trying to keep the exact same setup as before but it doesn't need to be inlined. Removed.